mos_tests/nova/nova_test.py

```diff
@@ -601,3 +606,80 @@ def test_boot_instance_from_volume_bigger_than_flavor_size(self):
         # Check that instance is reachable
         ping = common_functions.ping_command(self.floating_ip.ip)
         self.assertTrue(ping, "Instance after creation is not reachable")
+
+
+class TestNovaDeferredDelete(TestBase):
+    """Nova Deferred Delete test cases"""
+
+    @classmethod
+    @pytest.fixture(scope='class', autouse=True)
```

Comment:
When you will run all nova tests with `-S <snapshot>` argument result will be like this:
```
mos_tests/nova/nova_test.py::TestNovaDeferredDelete::test_1
mos_tests.nova.nova_test:617: ~~ SETUP ~~
mos_tests.nova.nova_test:626: test_1
PASSED
mos_tests.conftest: - revert snapshot <snapshot>
mos_tests/nova/nova_test.py::TestNovaDeferredDelete::test_2 ERROR
mos_tests/nova/nova_test.py::TestNovaDeferredDelete::test_3 ERROR
````